mindmeld/app_manager.py

```diff
@@ -98,14 +105,22 @@ def __init__(
 
         self.nlp = nlp or NaturalLanguageProcessor(app_path, resource_loader)
         self.question_answerer = question_answerer or QuestionAnswerer(
-            app_path, resource_loader, es_host
```

Comment:
Why are we not passing the es_host here?

Reply:
I am thinking of removing es_host from ApplicationManager's init param because it is not being used.

Reply:
On second thought I don't think we need this change in this PR, I will create an issue to keep track of it.